runtime/tutor.txt

```diff
@@ -140,8 +140,9 @@ _________________________________________________________________
     terminal.
  2. Open a file in Helix by running: hx FILENAME
  3. Make some edits to the file.
- 4. Press the : key to enter command mode.
- 5. Type w or write, and press <ENTER> to save the file.
+ 4. Press <ESC> to enter Normal mode.
```

Comment:
I don't see why this is necessary given that we teach them how to go back to normal mode. Maybe we can just say `In Normal mode, press the : ...`?

Reply:
Yeah, and by this point it's already been taught that keys are typed in insert mode, and do special actions in normal mode. Additionally, "make some edits" doesn't necessarily mean insertion; they can also delete text as has been taught by this point.